hack/RELEASE.md

```diff
@@ -0,0 +1,77 @@
+(FIXME: Should this file live elsewhere?)
+
+# Libpod/Podman/Podman-Remote Automated Release Workflow
+
+1. Open a new PR.  The the `HEAD` commit *must* have a summary line matching the
```

Comment:
It's probably not going to be HEAD but HEAD~1 - the format is Release commit, then -dev release commit (bumping to a fresh development version)

Reply:
Oh! Thanks.  Does the `-dev` release need to happen in the exact same PR or could it come after?  Does automation need to do anything in support of a `-dev` release?

So it goes (in order), and all in a single PR:

1. Release notes commit
2. Release commit
3. Dev Release commit

Reply:
oh nvm, I see you answered me blow...

Reply:
Duplicate 'the'